dockerfile/avalanche/Dockerfile

```diff
@@ -117,16 +114,10 @@ LABEL org.opencontainers.image.source="https://github.com/strangelove-ventures/h
 
 WORKDIR /bin
 
-# Install ln (for making hard links), rm (for cleanup), mv, mkdir amd vi from full busybox image (will be deleted, only needed for image assembly)
-COPY --from=busybox-full /bin/ln /bin/rm /bin/mv /bin/mkdir /bin/vi ./
-
 # Install minimal busybox image as shell binary (will create hardlinks for the rest of the binaries to this data)
 COPY --from=infra-toolkit /busybox/busybox /bin/sh
 
-# Install jq
-COPY --from=infra-toolkit /usr/local/bin/jq /bin/
```

Comment:
`jq` is not a busybox provided binary. We build it as a static binary in infra-toolkit, so it does need to be copied over separately like this

https://github.com/strangelove-ventures/infra-toolkit/blob/69a11239e5fcd51b75306d88f1c23fa8b59ee550/Dockerfile#L52